src/App.tsx

```diff
@@ -28,7 +29,12 @@ const App = () => {
     }
   })
 
-  return <Box className="bg">{userId ? <PrivateRouts /> : <PublicRoutes />}</Box>
+  return (
+    <ContextProvider>
```

Comment:
What you mean by name ContextProvider - Your context should have correct name!

Reply:
@gor017

Reply:
done

Reply:
Not fixed